.gitignore

```diff
@@ -15,3 +15,5 @@ airportdata/
 
 *.swp
 *.swo
+
+public/assets/bundle.js
```

Comment:
yeah I know we have to do other stuff to fix this but I didn't feel like it right now